components/cache/adapters/redis_adapter.rst

```diff
@@ -201,6 +201,17 @@ In order to use tag-based invalidation, you can wrap your adapter in :class:`Sym
     $client = RedisAdapter::createConnection('redis://localhost');
     $cache = new RedisTagAwareAdapter($client);
 
+Configuring Redis
+~~~~~~~~~~~~~~~~~
+
+When redis is used as cache its recommended to configure the `maxmemory` and `maxmemory-policy`
+read more about this topic in the offical `Redis LRU Cache Documentation`_. Normally the redis
+config can be found int `/etc/redis/redis.conf`. The `maxmemory` depend on the usage of the cache
```

Comment:
```suggestion
By setting `maxmemory`, you limit how much memory Redis is allowed to consume. If the amount is too low, Redis will drop entries that would still be useful and you benefit less from your cache. Setting the `maxmemory-policy` to 
`allkeys-lru` tells Redis that it is ok to drop data when it runs out of memory, and to first drop the oldest entries (least 
recently used). If you do not allow Redis to drop entries, it will return an error when you try to add data when no 
memory is available. An example setting could look as follows:
```